### PR TITLE
Increased in-cloud convective scavenging coefficients

### PR DESCRIPTION
**Pull request by oyvindseland**
Increased in-cloud scavenging coefficent for accumulation and coarse mode sea-salt, and aqueous phase produced sulfate.
Same changes as in the Forces version but modified so it use the same structure and syntax as the restructured Oslo-Aero.

### CODE REVIEW

File: src/oslo_aero_depos.F90 [hunk 1]
```diff
@@ -717,6 +718,12 @@ subroutine oslo_aero_depos_wet ( lchnk, ncol, psetcols, pmid, pdel, q, t, &
                 jnv = 0  !==> below cloud scavenging coefficients are zero (see above)
              endif
 
+! OS 240207 Increase scavenging efficiency for large soluble particles.
```
Comment (gold2718) [inline src/oslo_aero_depos.F90]: `git blame` does a good job of showing who made a change and when, we don't really need these inserts.
```suggestion
! Increase scavenging efficiency for large soluble particles.
```

File: src/oslo_aero_depos.F90 [hunk 2]
```diff
@@ -717,6 +718,12 @@ subroutine oslo_aero_depos_wet ( lchnk, ncol, psetcols, pmid, pdel, q, t, &
...
+              if (lphase==1.and.mm.eq.l_ss_a2.or.mm.eq.l_ss_a3.or.mm.eq.l_so4_a2) then
```
Comment (gold2718) [inline src/oslo_aero_depos.F90]: This seems like odd logic. Is it okay to make these changes even if `lphase` is _not_ `1`? That what is what you have. If `lphase` must be `1`, see this alternative.
```suggestion
              if ((lphase==1).and.((mm==l_ss_a2).or.(mm==l_ss_a3).or.(mm==l_so4_a2))) then
```

**Review by gold2718 — CHANGES_REQUESTED** [review]
I'm not sure about the logic and can submit my suggestions as a PR to your branch if you want.

**Comment by gold2718** [comment]
@oyvindseland, have you had a chance to look at my suggestions and questions?

**Comment by oyvindseland** [comment]
Hi. Yes I agree with your suggestions. Updated the commit.

**Review by gold2718 — APPROVED** [review]
Looks good now!

**Pull request merged**